examples/with-react-native-web/packages/ui/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "ui",
+  "version": "0.0.0",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "build": "tsup src/index.tsx --format esm,cjs --dts --external react",
+    "dev": "tsup src/index.tsx --format esm,cjs --watch --dts --external react",
+    "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf dist"
```

Comment:
`node_modules` will do nothing. This is the only place where you drop `.turbo` as well which we shouldn't recommend.

Reply:
Cleaned this to only remove dist